src/lib.rs

```diff
@@ -47,12 +57,23 @@ use wasm_bindgen::prelude::*;
 #[cfg(feature = "rust")]
 pub async fn create_graph(
   root_specifier: ModuleSpecifier,
-  loader: Box<dyn Loader>,
-  maybe_resolver: Option<Box<dyn Resolver>>,
+  loader: &mut dyn Loader,
```

Comment:
One clippy error lead to another and I ended up doing this. I think it might be better because then callers can keep holding onto the loader and resolver in case they need it.